katana/views/index.html

```diff
@@ -54,7 +54,7 @@
             </a>
             <a target="_blank" href="https://github.com/warriorframework/warriorframework">
                 <div class="pull-right big-navbar-icons tooltip-on-right-dark">
-                    <i class="fa fa-bitbucket fa-2x" aria-hidden="true"></i>
+                    <i class="fa fa-github fa-2x" aria-hidden="true"></i>
                     <span class="tooltip-content">Warrior Repository</span>
                 </div>
                 <div class="pull-right small-navbar-icons tooltip-on-right-dark">
```

Comment:
What is the line below for? it mentions bitbucket, also L32 has something we don't want